runtime/src/iree/vm/test/BUILD

```diff
@@ -59,167 +59,239 @@ c_embed_data(
 
 iree_bytecode_module(
     name = "assignment_ops",
     src = "assignment_ops.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "assignment_ops_f32",
     src = "assignment_ops_f32.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "assignment_ops_i64",
     src = "assignment_ops_i64.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "buffer_ops",
     src = "buffer_ops.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "call_ops",
     src = "call_ops.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "comparison_ops",
     src = "comparison_ops.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "comparison_ops_f32",
     src = "comparison_ops_f32.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "comparison_ops_i64",
     src = "comparison_ops_i64.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
 )
 
 iree_bytecode_module(
     name = "control_flow_ops",
     src = "control_flow_ops.mlir",
-    flags = ["--iree-vm-ir-to-bytecode-module"],
-    translate_tool = "//iree/tools:iree-translate",
+    flags = [
+        "--compile-mode=vm",
+        "--output-format=vm-c",
+    ],
+    translate_tool = "//iree/tools:iree-compile",
```

Comment:
This will be `//tools:iree-compile` after https://github.com/google/iree/pull/9135